build.sbt

```diff
@@ -0,0 +1,145 @@
+import com.typesafe.tools.mima.core._, ProblemFilters._
+import com.typesafe.tools.mima.plugin.MimaPlugin.mimaDefaultSettings
+import com.typesafe.tools.mima.plugin.MimaKeys.{
+  mimaBinaryIssueFilters,
+  mimaPreviousArtifacts
+}
+
+import sbtrelease.ReleasePlugin.autoImport._
+import sbtrelease.ReleaseStateTransformations._
+
+import xerial.sbt.Sonatype._
+
+lazy val bcSettings: Seq[Setting[_]] = mimaDefaultSettings ++ Seq(
+  mimaPreviousArtifacts := {
+    val sv = scalaVersion.value
+
+    if (sv startsWith "2.9.") {
+      Set("com.jsuereth" % "scala-arm_2.9.1" % "1.2")
+    } else if (sv startsWith "2.10.") {
+      Set("com.jsuereth" %% "scala-arm" % "1.2")
+    } else if (sv startsWith "2.13.") {
+      Set.empty[ModuleID]
+    } else {
+      Set("com.jsuereth" %% "scala-arm" % "2.0")
+    }
+  },
+  mimaBinaryIssueFilters ++= {
+    val missingMethodInOld: ProblemFilter = {
+      case ReversedMissingMethodProblem(_) => false
+      case DirectMissingMethodProblem(old) => !old.isAccessible
+      case InheritedNewAbstractMethodProblem(_, _) => false
+      case IncompatibleResultTypeProblem(old, _) => !old.isAccessible
+      case IncompatibleMethTypeProblem(old, _) => !old.isAccessible
+      case MissingClassProblem(old) => !old.isPublic
+      case AbstractClassProblem(old) => !old.isPublic
+      case _ => true
+    }
+
+    Seq(
+      missingMethodInOld, // forward compat
+      ProblemFilters.exclude[IncompatibleTemplateDefProblem]("resource.AbstractManagedResource"), // previous change
+      ProblemFilters.exclude[MissingTypesProblem]("resource.DefaultManagedResource"), // previous change
+      ProblemFilters.exclude[UpdateForwarderBodyProblem]("resource.ManagedResourceOperations.toTraversable") // mixin for 2.13 compat
+    )
+  }
+)
+
+lazy val publishSettings: Seq[Setting[_]] = Seq(
```

Comment:
!! Without credentials, I cannot check the migration of the publish/site/jekyll settings